nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -372,7 +372,7 @@ def restore_from(
         loaded_params = super().load_config_and_state_dict(
             calling_cls, restore_path, override_config_path, map_location, strict, return_config, trainer,
         )
-        if not isinstance(loaded_params, tuple):
+        if not isinstance(loaded_params, tuple) or return_config is True:
```

Comment:
```suggestion
        if not isinstance(loaded_params, tuple) or return_config:
```

Could we not just simplify to this? Seems it's always a bool

Reply:
I prefer explicit checks generally, even for bool assertion